### PR TITLE
Update branding to 3.1.13

### PR DESCRIPTION
**Pull request by wtgodbe**
And update SDK

### CODE REVIEW

**Review by JunTaoLuo — APPROVED** [review]

File: NuGet.config [hunk 1]
```diff
@@ -17,5 +24,23 @@
     <!--  Begin: Package sources from dotnet-corefx -->
     <add key="darc-int-dotnet-corefx-59d2f36" value="true" />
     <!--  End: Package sources from dotnet-corefx -->
+    <add key="darc-int-dotnet-extensions-bfff66d" value="true" />
```
Comment (dougbu) [inline NuGet.config:27]: @mmitche @MattGal I thought the Maestro++ bug around leaving behind extra feed suppressions had been fixed❔

Comment (wtgodbe) [reply NuGet.config:27]: I fixed up some nuget.configs manually last month, this might have been me

Comment (mmitche) [reply NuGet.config:27]: This is just the merge to open, so it is the commit state prior to @MattGal's fix I think.

Comment (MattGal) [reply NuGet.config:27]: @dougbu the latest changes should never delete comments and honor the order they're in (even if they're not alphabetical).  I'll come back when there is a dependency flow that needs it though.

Comment (dougbu) [reply NuGet.config:27]: Thanks @wtgodbe @mmitche and @MattGal 

My questions were also influenced by dotnet/aspnetcore#30058. In particular,
- the weird placement of an end comment at https://github.com/dotnet/aspnetcore/pull/30058/files#diff-9c5952957709545aad811b400e7e516eebade1e44fc4fbc23203403ffeb92c34R39
- the added duplicate lines at https://github.com/dotnet/aspnetcore/pull/30058/files#diff-9c5952957709545aad811b400e7e516eebade1e44fc4fbc23203403ffeb92c34R40 and https://github.com/dotnet/aspnetcore/pull/30058/files#diff-9c5952957709545aad811b400e7e516eebade1e44fc4fbc23203403ffeb92c34R45

Thoughts❔

**Pull request merged**